content/docs/3.0/faq/faq.md

```diff
@@ -0,0 +1,166 @@
+---
+title: FAQ
+description: FAQs about TiKV
+menu:
+    nav:
+        parent: Docs
+        weight: 5
+    docs:
+        weight: 5
+---
+
+## What is TiKV?
+
+TiKV is a distributed Key-Value database that features in geo-replication, horizontal scalability, consistent distributed transactions and coprocessor support. 
+
+## How do TiDB and TiKV work together? What is the relationship between the two?
+
+TiDB works as the SQL layer and TiKV works as the Key-Value layer. TiDB provides TiKV the SQL enablement and turns TiKV into a NewSQL database. TiDB and TiKV work together to be as scalable as a NoSQL database while maintains the ACID transactions of a relational database.
+
+## Why do you have separate layers?
+
+Inspired by Google F1 and Spanner, TiDB and TiKV adopt a highly-layered architecture. This architecture supports pluggable storage drivers and engines, which powers you to customize your database solutions based on your own business requirements. Meanwhile, this architecture makes it easy to debug, update, tune, and maintain. You won’t have to go through the whole system just to find and fix a bug in one module.
+
+## How do I run TiKV?
+
+For further information, see [Quick Start](../tasks/getting-started.md) for deploying a TiKV testing cluster, or [Deploy TiKV](../tasks/deploy/introduction.md) for deploying TiKV in production.
+
+## When to use TiKV?
+
+TiKV is at your service if your applications require:
+
+* Horizontal scalability (including writes)
+* Strong consistency
+* Support for distributed ACID transactions
+
+## When not to use TiKV?
+
+TiKV is not yet ready to deal with very low latency reads and writes.
+
+## How does TiKV scale?
+
+Grow TiKV as your business grows. You can increase the capacity simply by adding more machines. You can run TiKV across physical, virtual, container, and cloud environments.
+
+PD([Placement Driver](https://github.com/pingcap/pd))  periodically checks replication constraints and balances the load, and it handles data movement automatically. When PD notices that the load is too high, it will rebalance data.
```

Comment:
PD ([Placement Driver](https://github.com/pingcap/pd)) periodically checks replication constraints and balances the load, and it handles data movement automatically. When PD notices that the load is too high, it will rebalance data.
```suggestion
PD ([Placement Driver](https://github.com/pingcap/pd)) periodically checks replication constraints and balances the load, and it handles data movement automatically. When PD notices that the load is too high, it will rebalance data.
```